src/PortingAssistantExtensionServer/PortingAssistantExtensionServer.csproj

```diff
@@ -3,16 +3,18 @@
 	<PropertyGroup>
 		<OutputType>Exe</OutputType>
 		<TargetFramework>netcoreapp3.1</TargetFramework>
+		<IncludeSymbolsInSingleFile>true</IncludeSymbolsInSingleFile>
 	</PropertyGroup>
 
 	<PropertyGroup>
 		<RestorePackagesWithLockFile>true</RestorePackagesWithLockFile>
 	</PropertyGroup>
 
 	<ItemGroup>
+		<PackageReference Include="LibGit2Sharp" Version="0.26.2" />
```

Comment:
Do we still need this direct dependency?

Reply:
Oh probably not, I'll remove that